Include/httpClient/config.h

```diff
@@ -6,7 +6,6 @@
 #define HC_PLATFORM_UWP 2
 #define HC_PLATFORM_XDK 3
 #define HC_PLATFORM_GDK 4
-#define HC_PLATFORM_GRTS 5
```

Comment:
>HC_PLATFORM_GRTS [](http://example.com/codeflow?start=8&length=16)

is this not needed for backward compat anymore?

Reply:
we only ever used this internally in grts itself and we do not use it anymore